src/layer.rs

```diff
@@ -237,7 +237,15 @@ impl LayerContents {
     where
         F: FnMut(&Layer) -> bool,
     {
-        self.layers.retain(|layer| layer.name == DEFAULT_LAYER_NAME || predicate(layer))
+        // Always keep the default layer, which is always the first.
```

Comment:
who says that a default layer has to be the first one? Default layer is simply the one that points to the default "glyphs" folder, it can have any name ("public.default" is simply the name for a default layer without an explicit name and is not sufficient to mark a layer as "default" ) and can be in any position not necessarily the first. Unless you reordered the layers (which you should have not).

Reply:
> Unless you reordered the layers (which you should have not).

oh I see you actually do reorder the layers:

https://github.com/linebender/norad/blob/0b9e43334551861dbdd75732785d0cb5f3431568/src/layer.rs#L88-L93

I think this is incorrect, the UFO spec never says the default should be the first.

Reply:
Filed https://github.com/linebender/norad/issues/343